packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -114,15 +114,18 @@ class PreviewCommand extends ShorebirdCommand {
       return ExitCode.success.code;
     }
 
-    final releases = await codePushClientWrapper.getReleases(
-      appId: appId,
-      sideloadableOnly: true,
-    );
+    final (allReleases, sideloadableReleases) = await (
+      codePushClientWrapper.getReleases(appId: appId),
+      codePushClientWrapper.getReleases(
+        appId: appId,
+        sideloadableOnly: true,
+      )
+    ).wait;
```

Comment:
Why not just make a single request and filter for `isSideloadable`? That seems like it'd be a lot more efficient.

Reply:
Explained on VC. I'd leave a comment here to explain why we're doing this.